src/frontend/src/lib/styles/global/button.scss

```diff
@@ -167,4 +168,9 @@ button {
 			}
 		}
 	}
+
+	&.about {
+		background: rgba(0, 0, 0, 0.3);
```

Comment:
nth: black and white colors are defined as css variable as rgb, can we reuse those variable here too?

Reply:
ah right! however, I cannot find a way to adjust the opacity of those variables. I tried a few things and didn't work, for example

1. rgba(var(--color-black), 30%)
2. mixin color.with-opacity

Reply:
Thanks for trying. Indeed there are two issues preventing us to do so.

- Unlike white, black is not redeclared as a variable so `--color-black` is unknown 😅
- Even if it would be know it would be set as rgb so we can tweak it, it would render to `rgba(rgb(0,0,0),0.4)`

Alright so let's forget about it.